openedx/features/discounts/models.py

```diff
@@ -0,0 +1,36 @@
+"""
+DiscountRestrictionConfig Models
+"""
+
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
+
+from django.utils.encoding import python_2_unicode_compatible
+
+from openedx.core.djangoapps.config_model_utils.models import StackedConfigurationModel
+
+
+@python_2_unicode_compatible
+class DiscountRestrictionConfig(StackedConfigurationModel):
+    """
+    A ConfigurationModel used to manage restrictons for lms-controlled discounts
+    """
+
+    STACKABLE_FIELDS = ('enabled',)
+
+    @classmethod
+    def disabled_for_course(cls, course):
+        """
+        Return whether lms-controlled discounts can be enabled for this course.
+        Checks if discounts are disabled for attributes of this course like Site, Partner, Course or Course Run.
+
+        Arguments:
+            course: The CourseOverview of the course being queried.
+        """
+        current_config = cls.current(course_key=course.id)
+        return not current_config.enabled
```

Comment:
enabled is a field on stacked config models, disabled is not.  
i suppose i could make a function rather than a field in stackedconfigmodels called disabled that returns not enabled.  
changing how stackedconfigmodels work for this seems suboptimal to me, but if you feel strongly about it I will change it